notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-760_Analysis.ipynb

```diff
@@ -0,0 +1,1217 @@
+   "id": "00bbb5e2-748e-493a-b3ec-fd4d8df4a45e",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-07-05T15:35:25.285740Z",
+     "iopub.status.busy": "2023-07-05T15:35:25.285468Z",
+     "iopub.status.idle": "2023-07-05T15:35:25.333356Z",
+     "shell.execute_reply": "2023-07-05T15:35:25.332560Z",
+     "shell.execute_reply.started": "2023-07-05T15:35:25.285724Z"
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import itertools as itt\n",
+    "import pandas as pd\n",
+    "import matplotlib.dates as mdates\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib.ticker import FormatStrFormatter\n",
+    "import numpy as np\n",
+    "import seaborn as sns\n",
+    "\n",
+    "from astropy import units as u\n",
+    "from astropy.time import Time, TimezoneInfo\n",
+    "\n",
+    "from lsst.sitcom import vandv\n",
+    "from lsst.ts.idl.enums import MTM1M3"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "d933c046-a1a2-4d66-96a0-8a6b46bff479",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-07-05T12:56:30.137985Z",
+     "iopub.status.busy": "2023-07-05T12:56:30.137722Z",
+     "iopub.status.idle": "2023-07-05T12:56:31.322100Z",
+     "shell.execute_reply": "2023-07-05T12:56:31.321505Z",
+     "shell.execute_reply.started": "2023-07-05T12:56:30.137969Z"
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Create client to access the EFD database\n",
+    "client = vandv.efd.create_efd_client()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 73,
+   "id": "361fe8dc-295b-4e01-a44a-363a8ab1b428",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-07-05T14:59:47.025364Z",
+     "iopub.status.busy": "2023-07-05T14:59:47.025086Z",
+     "iopub.status.idle": "2023-07-05T14:59:47.088498Z",
+     "shell.execute_reply": "2023-07-05T14:59:47.087918Z",
+     "shell.execute_reply.started": "2023-07-05T14:59:47.025339Z"
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "pos_columns = [a + \"Position\" for a in \"xyz\"]\n",
+    "rot_columns = [a + \"Rotation\" for a in \"xyz\"]\n",
+    "all_columns = pos_columns + rot_columns"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 78,
+   "id": "c5560a0e-dc1f-403f-a5c4-501133fd0c19",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-07-05T15:24:00.676310Z",
+     "iopub.status.busy": "2023-07-05T15:24:00.676041Z",
+     "iopub.status.idle": "2023-07-05T15:24:07.814076Z",
+     "shell.execute_reply": "2023-07-05T15:24:07.813552Z",
+     "shell.execute_reply.started": "2023-07-05T15:24:00.676293Z"
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# get IMS data\n",
+    "df_ims = await client.select_time_series(\n",
+    "    \"lsst.sal.MTM1M3.imsData\", \n",
+    "    \"*\", \n",
+    "    Time(t_start, format=\"isot\", scale=\"utc\"),\n",
```

Comment:
You can spare some time by assigning t_start = Time(...) and use it in calls as Time object, don't do the conversion all the time